projects/js-packages/publicize-components/src/social-store/types.ts

```diff
@@ -132,6 +132,11 @@ export type UtmSettingsConfig = {
 	enabled: boolean;
 };
 
+export type SocialNotesConfig = {
+	append_link: boolean;
+	link_format: 'full_url' | 'shortlink' | 'permashortcitation';
+};
+
```

Comment:
Can we please update the `SocialSettingsFields` type down there with the appropriate keys like
```ts
...
jetpack_social_notes_config: SocialNotesConfig;
...

Likewise, the other keys we added here

Reply:
Fixed in 44f8c61, thanks!